charts/hivemq-platform/values.schema.json

```diff
@@ -15,7 +15,11 @@
           "type" : "string"
         },
         "overrideStatefulSet" : {
-          "type" : "string"
+          "type": "string",
+          "contentMediaType": "application/json",
+          "contentSchema": {
+            "$ref": "https://raw.githubusercontent.com/kubernetes/kubernetes/v1.31.1/api/openapi-spec/swagger.json#/definitions/io.k8s.api.apps.v1.StatefulSet"
```

Comment:
What happens if we have a non-compatible update for one of these definitions between lets say k8s 1.25 and 1.31. If the customer still runs k8s 1.25 that should be fine. But we would reject that or?

Reply:
Already agreed to discuss this later on.